spec/features/toc_spec.rb

```diff
@@ -0,0 +1,27 @@
+# frozen_string_literal: true
+
+require "rails_helper"
+
+describe "Table of Contents", type: :feature, js: true do
```

Comment:
I understand you decided having a TOC feature spec by itself is a good idea. I can get behind that.

If so can we move the other toc specs here?

https://github.com/pulibrary/pulfalight/blob/main/spec/features/catalog_view_spec.rb#L118-L120

https://github.com/pulibrary/pulfalight/blob/main/spec/features/catalog_view_spec.rb#L91-L94

https://github.com/pulibrary/pulfalight/blob/main/spec/features/catalog_view_spec.rb#L319-L324

Reply:
Done.